### ISSUE TITLE
Composer does not work anymore

### ISSUE THREAD

**Issue by MatthiasBrodelet** [lando/lando#3625]
I have been using Lando for years now without too much issues but now I seem to have hit a rather weird bug.
When running `lando composer install` I get the following message:
_You must set up the project dependencies using `composer install`
See https://getcomposer.org/download/ for instructions on installing Composer_

This is my lando file:
```
name: the-house-webshop
recipe: lamp
config:
    webroot: web
    php: 8.1
    database: mysql

services:
    database:
        type: mysql
    pma:
        type: phpmyadmin
        hosts:
            - database
```
            
I have uninstalled and reinstalled Lando and Docker. I also downgraded both Lando and docker but the issue persists.
I'm running on MacOS with M1 Pro.

What is the cause of this issue and how can it be fixed?

Update: It seems that `lando composer` only works when there is a composer.lock file. This seems kind of annoying because all I wanted to do was `lando composer install`

**Comment by aasarava** [comment]
Try deleting your /vendor directory and then running `lando composer install`. Worked for me.

**Comment by morovinger** [comment]
same here

**Comment by morovinger** [comment]
> Try deleting your /vendor directory and then running `lando composer install`. Worked for me.

same here. This dosent help at all, now i get Warning: include(/app/bin/../vendor/composer/composer/bin/composer): Failed to open stream: No such file or directory in /app/bin/composer on line 119

**Comment by bronius-congruity** [comment]
I agree that removing `./vendor` can fix all regular hiccups, but I think the original poster and additional commenter(s) may have experienced what I just did: the command `lando composer install` outputs nothing. `lando drush st` also stopped being recognized. I did just update lando from `v3.21.2` to `v3.23.7` and was stepped through some "update/install all these plugins?" to which of course I said yes. I also had to run `lando setup` which sent me in a loop, but `lando setup --clear` seems to have got me into a working lando again.

What can I provide to help diagnose? Or better, recover? :)

I'll open a new support thread if this is not the same as what the original commenters posted about. Thanks.
